### PR TITLE
Closes #131 - featured team on userprofile

### LINKED ISSUE #131: Userprofile - allow users to specify a specific team to showcase on their profile
add an additional fk field to userprofile to hold said value, and a form field in edit_profile that allows a user to change the featured team

### PR DESCRIPTION
**Pull request by mikemaddem**

### CODE REVIEW

File: profiles/forms.py [hunk 1]
```diff
@@ -36,38 +36,16 @@ class Meta:
...
-        self.fields['favorite_game'].widget.attrs.update({'name': 'favorite_game', 'class': 'form-control', 'style':
-            ''})
-        self.fields['favorite_console'].widget.attrs.update(
-            {'name': 'favorite_console', 'class': 'form-control', 'style':
-                ''})
-        self.fields['country'].widget.attrs.update({'name': 'country', 'class': 'form-control'})
-        self.fields['email_enabled'].widget.attrs.update({'name': 'email_enabled', 'class': 'form-control',
-                                                          'style': 'background-color: black;'})
+        self.fields['feat_team'].queryset = self.instance.founder_teams.all() | self.instance.captain_teams.all() | self.instance.player_teams.all()
+        # super(EditProfileForm, self).__init__(*args, **kwargs)
```
Comment (dannygreg) [inline profiles/forms.py:46]: Should probably remove this commented out code.

File: profiles/models.py [hunk 2]
```diff
@@ -94,6 +94,8 @@ def __str__(self):
     country = CountryField(blank_label='(select country)', default='US')
 
     email_enabled = models.BooleanField(default=True)
+    # team to display on their profile
+    feat_team = models.ForeignKey('teams.Team', related_name='profile_feat_team', blank=True, null=True, on_delete=models.SET_NULL)
```
Comment (dannygreg) [inline profiles/models.py:98]: It's a nitpick - but I don't usually like abbreviated variable names. We don't pay-by-the-byte these days so may as well make it `featured_team`.

**Review by dannygreg — CHANGES_REQUESTED** [review]
Only a couple nits really.

**Comment by mikemaddem** [comment]
@dannygreg  @techlover1 Changes made

**Pull request merged**